packages/storybook8/stories/Concepts/RealTimeText/Docs.mdx

```diff
@@ -0,0 +1,62 @@
+import { Stack, PrimaryButton } from '@fluentui/react';
+import { Meta, Source } from '@storybook/addon-docs';
+import { overviewPageImagesStackStyle } from '../../constants';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
+
+<Meta title="Concepts/Real-Time Text" />
+
+# Real-Time Text
+
+<SingleLineBetaBanner />
+
+Azure Communication Services UI Library is adding support for Real-Time Text. Real-Time Text is a powerful tool that enables developers to enhance the accessibility of their videos.
+Real-Time Text (RTT) is a new accessibility feature that enables live transmission of text as it is typed, enhancing communication during voice and video calls. RTT displays text instantly, character by character, creating a natural and dynamic communication experience that mirrors spoken conversation.
+
+Real-Time Text is supported by default and are automatically included within the CallComposite and CallWithChatComposite experiences.
+
+Real-Time Text is also supported for component only users.
+
+List of components exported for Real-Time Text:
+
+[RealTimeText](./?path=/docs/components-real-time-text--docs) is a single line of Real Time Text Message. It will display an 'isTyping' badge when the real time text message is still ongoing.
+
+[RealTimeTextModal](./?path=/docs/components-real-time-text-modal--docs) is a component showing a disclaimer warning that real time text is enabled for everyone in the call once turned on. This is important as real time text can not be turned off once it is started.
+
+[StartRealTimeTextButton](./?path=/docs/components-start-real-time-text-button--docs) is a component that can be used to turn on real time text. Note that Real-Time Text will not be started for everyone in the call until a message is sent.
```

Comment:
how come Real-Time text here is spelt like this here but no where else?

Reply:
* correction, we use `real-time text` here and some other places. But we're mixing in a few `real time text` around. Can we stick to just `real-time text`